data/targets.yml

```diff
@@ -13,6 +23,16 @@
   obs_project: "Devel:YaST:Head"
   obs_sr_project: "SUSE:SLE-12-SP2:GA"
   obs_target: "SLE-12-SP1"
+:sle12sp3:
+  obs_api: "https://api.suse.de/"
+  obs_project: "Devel:YaST:Head"
+  obs_sr_project: "SUSE:SLE-12-SP3:GA"
+  obs_target: "sle12_sp3"
+:sle_latest:
```

Comment:
yeap, I know it, but I try to be KISS,  I know it is not DRY, but it is obvious where we send what.